packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -0,0 +1,164 @@
+import { expect } from 'chai';
+import semver from 'semver';
+import type { CompassBrowser } from '../helpers/compass-browser';
+import { beforeTests, afterTests } from '../helpers/compass';
+import type { Compass } from '../helpers/compass';
+import { MONGODB_VERSION } from '../helpers/compass';
+import * as Selectors from '../helpers/selectors';
+import { createDummyCollections } from '../helpers/insert-data';
+
+describe('FLE2', function () {
+  let initialEnvVars: NodeJS.ProcessEnv;
+
+  before(async function () {
+    if (
+      semver.lt(MONGODB_VERSION, '6.0.0-rc0') ||
+      process.env.MONGODB_USE_ENTERPRISE !== 'yes'
+    ) {
+      return this.skip();
+    }
+
+    initialEnvVars = Object.assign({}, process.env);
+    process.env.COMPASS_CSFLE_SUPPORT = 'true';
```

Comment:
I'm not sure about this .. I have another comment where we set `COMPASS_CSFLE_SUPPORT` in the connection form tests. We decided not to enable FF tests in CI, maybe we need a way to run them in some instances (we should be able to do that with `evergreen patch`). 

The idea is that we need the CI to test the real state of the code that we would be releasing, so to give us the confidence nothing is broken.

For this PR if you enable the FF right after, I think we can keep it, but bear in mind for next time that normally we don't want to enable FF on CI by default.